src/dymaptic.GeoBlazor.Core/Components/PortalFolder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components;
+public class PortalFolder : MapComponent
+{
+
+    /// <summary>
+    ///    The unique id of the folder.
+    /// </summary>
+    public new string Id { get; set; } = default!;
+
+    /// <summary>
+    /// The date the folder was created.
+    /// </summary>
+    public DateTime CreatedDate { get; set; }
+
+
+    /// <summary>
+    ///  The portal associated with the folder.
+    /// </summary>
+    public Portal Portal { get; set; } = default!;
```

Comment:
As a `MapComponent`, this would be assumed to work with nested components, so simple properties should be `[Parameter]` and `Portal` would be added via `RegisterChildComponents`.